packages/grid_client/src/client.ts

```diff
@@ -245,6 +247,8 @@ class GridClient {
       substrate: substrateURL || `wss://tfchain.${base}/ws`,
       graphql: graphqlURL || `https://graphql.${base}/graphql`,
       activation: activationURL || `https://activation.${base}/activation/activate`,
+      //TODO handle kyc url
```

Comment:
remove this comment

Reply:
Needed to be handled later, on supporting kyc stacks